src/ports/collections/queries.ts

```diff
@@ -0,0 +1,98 @@
+import SQL, { SQLStatement } from 'sql-template-strings'
+import { MARKETPLACE_SQUID_SCHEMA } from '../../constants'
+import { getWhereStatementFromFilters } from '../utils'
+import { CollectionFilters, CollectionSortBy } from './types'
+
+function getCollectionsSortByStatement(filters: CollectionFilters): SQLStatement {
+  switch (filters.sortBy) {
+    case CollectionSortBy.NEWEST:
+      return SQL` ORDER BY created_at DESC `
+    case CollectionSortBy.RECENTLY_REVIEWED:
+      return SQL` ORDER BY reviewed_at DESC `
+    case CollectionSortBy.NAME:
+      return SQL` ORDER BY name ASC `
+    case CollectionSortBy.SIZE:
+      return SQL` ORDER BY items_count DESC `
+    case CollectionSortBy.RECENTLY_LISTED:
+      return SQL` ORDER BY first_listed_at DESC `
+    default:
+      return SQL` ORDER BY name ASC `
+  }
+}
+
+function getCollectionsLimitAndOffsetStatement(filters: CollectionFilters): SQLStatement {
+  const limit = filters?.first ? filters.first : 1000
+  const offset = filters?.skip ? filters.skip : 0
+
+  return SQL` LIMIT ${limit} OFFSET ${offset} `
+}
+
+function getCollectionsWhereStatement(filters: CollectionFilters): SQLStatement {
+  const FILTER_BY_CONTRACT_ADDRESS = filters.contractAddress ? SQL`LOWER(id) = ${filters.contractAddress.toLowerCase()}` : null
```

Comment:
This would follow the same as below:
```suggestion
  const FILTER_BY_CONTRACT_ADDRESS = filters.contractAddress ? SQL`id = ${filters.contractAddress.toLowerCase()}` : null
```